torch/testing/_internal/common_methods_invocations.py

```diff
@@ -17439,6 +17439,11 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
             DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_executor'),
         )
     ),
+    PythonRefInfo(
+        "_refs.nn.functional.group_norm",
```

Comment:
How thorough are the existing `nn.functional.group_norm` sample or reference inputs to validate this consistency? Are interesting edge cases covered? Do ErrorInputs need to be added?